client/blocks/plan-storage/style.scss

```diff
@@ -14,8 +14,8 @@
 .plan-storage__storage-label,
 .plan-storage__storage-link {
 	font-size: 11px;
-	height: 24px;
-	line-height: 24px;
+	line-height: 1;
```

Comment:
;) had to look up this one. https://css-tricks.com/almanac/properties/l/line-height/#article-header-id-0